lib/internal/streams/writable.js

```diff
@@ -428,11 +515,13 @@ function writeOrBuffer(stream, state, chunk, encoding, callback) {
 
   // stream._write resets state.length
   const ret = state.length < state.highWaterMark;
+
   // We must ensure that previous needDrain will not be reset to false.
-  if (!ret)
+  if (!ret) {
     state.state |= kNeedDrain;
+  }
```

Comment:
Nit: revert style change

Reply:
I think it's fine for ronag to make these sort of style changes tbh not a big deal